website/docs/layouts/flex/partials/code/how-to-use.md

```diff
@@ -0,0 +1,569 @@
+  <Doc::placeholder @width="300px" @height="40px" @background="#fff8d2">Extra content</Doc::placeholder>
+  <Doc::placeholder @width="240px" @height="40px" @background="#f3d9c5">Even more content</Doc::placeholder>
+</Hds::Layout::Flex>
+```
+In this case the `@wrap=\{{true}}` applied to the flex container allows the flex items to wrap on multiple lines. The `@gap=\{{array "8" "24"}}` applied to the flex container defines the gap between the items along the two "directions" (the first value is the space between "rows", the second between "columns").
+
+
+### Sizing
+
+In flexbox it's possible to control the relative sizing of flex items within a flex container using the `basis`, `grow`, and `shrink` properties.
+
+In the `Flex::Layout`, these properties translate to the `@basis`, `@grow`, and `@shrink` arguments.
+
+```handlebars{data-execute=false}
+<Hds::Layout::Flex as |LF|>
+  ...
+  <LF.Item @basis="200px" @grow={{1}} @shrink={{0}}>
+    {{! the flex item content here }}
+  </HLF.Item>
+  ...
+</Hds::Layout::Flex>
+```
+
+How these three properties impact the (relative) size of a flex item is not trivial, we suggest consulting [the MDN guide](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_flexible_box_layout/Basic_concepts_of_flexbox).
+
+#### Force items to use all the available space
+
+By default, flex items use only the space necessary to fit their content. If there is extra space in the flex container, the items will not automatically expand to fill it.
+
+```handlebars
+<Hds::Layout::Flex @gap="16" class="doc-flex-outlined-container" as |LF|>
+  <Doc::placeholder @width="auto" @height="40px" @background="#d2f4ff">Some content</Doc::placeholder>
+  <LF.Item>
+    <Doc::placeholder @height="40px" @background="#e5ffd2">This content occupies the minimum space possible</Doc::placeholder>
+  </LF.Item>
+  <Doc::placeholder @width="auto" @height="40px" @background="#d2f4ff">Extra content</Doc::placeholder>
+</Hds::Layout::Flex>
+```
+
+If we want an item to to grow and occupy the available space, we have to use its `grow` property.
+
+```handlebars
+<Hds::Layout::Flex @gap="16" class="doc-flex-outlined-container" as |LF|>
+  <Doc::placeholder @width="auto" @height="40px" @background="#d2f4ff">Some content</Doc::placeholder>
+  <LF.Item @grow={{true}}>
+    <Doc::placeholder @height="40px" @background="#e5ffd2">This content occupies as much space as possible</Doc::placeholder>
+  </LF.Item>
+  <Doc::placeholder @width="auto" @height="40px" @background="#d2f4ff">Extra content</Doc::placeholder>
+</Hds::Layout::Flex>
+```
+
+In this case we have applied `@grow=\{{true}}` to the wrapping `Flex::Item`, forcing it to use all the available space inside the flexbox container.
+
+#### Prevent collapsing of content
+
+In the example below, you can see (look at the button element) how, if one of the flex items has an intrinsic size larger than the flex container, it may squeeze the rest of the flex items:
```

Comment:
```suggestion
In the example below, you can see how, if one of the flex items has an intrinsic size larger than the flex container, it may squeeze the rest of the flex items (specifically the button element in this case).
```